build.sh

```diff
@@ -44,3 +41,10 @@ rm -f bazelisk
 ### Print some information about the generated binaries.
 ls -lh bin/*
 file bin/*
+
+# Non-googlers: you should run this script with NPM_REGISTRY=https://registry.npmjs.org
+readonly REGISTRY=${NPM_REGISTRY:-https://wombat-dressing-room.appspot.com}
+# Googlers: you should npm login using the go/npm-publish service:
+# $ npm login --registry https://wombat-dressing-room.appspot.com
+# TODO: also publish the release to npm.
+# ./bazelisk run --config=release //:npm_package.publish -- --access=public --tag latest --registry $REGISTRY
```

Comment:
Does the "TODO" mean that this is a manual step, or is there still something missing before we can publish to npm?

Reply:
It just needs some manual testing. I could do that myself, but maybe it's better for you or Florian to do the first release to make sure permissions are right.

Reply:
I removed the comment, but it means after merging this, you'll trip over any problems with npm publishing the next time you run it.

(Also I notice the `build.sh` script doesn't actually do a publish step yet, so maybe this isn't the right place for this command to be added? I didn't see any other release instructions where it belongs instead)